<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>10/18/1</td>
</tr><tr>
<td>Started on:</td><td>Thu Apr 06 07:39:55 IST 2023</td>
</tr>
<tr><td>Total time:</td><td>26 seconds (26299 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: sourav, john@testing.com, this is testdata5</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [446]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace894861974", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace894861974'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [446]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: vidya, john@testing.com, this is testdata7</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [889]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace735902909", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace735902909'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [889]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: john, john@testing.com, this is testdata9</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [416]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace984405229", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace984405229'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [416]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: john, john@testing.com, this is testdata9</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [754]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace876520952", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace876520952'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [754]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: paul, john@testing.com, this is testdata3</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [539]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace397733651", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace397733651'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [539]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: max, john@testing.com, this is testdata4</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [642]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace23761667", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace23761667'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [642]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: sourav, john@testing.com, this is testdata5</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [708]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1149480330", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1149480330'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [708]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: vaibhav, vaibhav@testing.com, this is testdata1</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [722]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace155382697", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace155382697'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [722]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: arun, john@testing.com, this is testdata6</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [813]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2058660095", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2058660095'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [813]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: abdul, john@testing.com, this is testdata8</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [1484]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace159361073", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace159361073'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [1484]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>1</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: john, john@testing.com, this is testdata2</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [829]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2004345444", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2004345444'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [829]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: vaibhav, vaibhav@testing.com, this is testdata1</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [1000] but found [477]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1111521172", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1111521172'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [1000] but found [477]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>1</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: arun, john@testing.com, this is testdata6</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [555]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1872404876", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1872404876'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [555]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: vidya, john@testing.com, this is testdata7</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [552]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace361585210", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace361585210'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [552]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC1_validAPIKeys()'><b>TC1_validAPIKeys</b><br>Test class: testcases.CreateCustomerAPI<br>Parameters: paul, john@testing.com, this is testdata3</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [730]
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace269200146", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace269200146'><pre>java.lang.AssertionError: The following asserts failed:
	expected [1000] but found [730]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.CreateCustomerAPI.TC1_validAPIKeys(CreateCustomerAPI.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: john, john@testing.com, this is testdata2</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [424]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1455031525", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1455031525'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [424]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: max, john@testing.com, this is testdata4</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [552]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace621233976", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace621233976'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [552]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC1_validAPIKey()'><b>TC1_validAPIKey</b><br>Test class: testcases.GetCustomerAPI<br>Parameters: abdul, john@testing.com, this is testdata8</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [1045]
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace484006022", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace484006022'><pre>java.lang.AssertionError: The following asserts failed:
	expected [400] but found [200],
	expected [HTTP/1.1 400 OK] but found [HTTP/1.1 200 OK],
	expected [vaibhav@testing.com] but found [john@testing.com],
	expected [1000] but found [1045]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testcases.GetCustomerAPI.TC1_validAPIKey(GetCustomerAPI.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>1</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: paul, john@testing.com, this is testdata3</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: max, john@testing.com, this is testdata4</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: john, john@testing.com, this is testdata9</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: sourav, john@testing.com, this is testdata5</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: arun, john@testing.com, this is testdata6</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: abdul, john@testing.com, this is testdata8</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: vaibhav, vaibhav@testing.com, this is testdata1</td>
<td></td>
<td>3</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: vidya, john@testing.com, this is testdata7</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='utility.DataExtractor.CreateCustomer()'><b>CreateCustomer</b><br>Test class: utility.DataExtractor<br>Parameters: john, john@testing.com, this is testdata2</td>
<td></td>
<td>0</td>
<td>utility.DataExtractor@56620197</td></tr>
<tr>
<td title='testcases.GetCustomerAPI.TC2_invalidAPIKey()'><b>TC2_invalidAPIKey</b><br>Test class: testcases.GetCustomerAPI</td>
<td></td>
<td>0</td>
<td>testcases.GetCustomerAPI@479d31f3</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testcases.CreateCustomerAPI.TC2_skipTestcase()'><b>TC2_skipTestcase</b><br>Test class: testcases.CreateCustomerAPI</td>
<td><div><pre>org.testng.SkipException: Test is Skipped
	at testcases.CreateCustomerAPI.TC2_skipTestcase(CreateCustomerAPI.java:79)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace410529367", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace410529367'><pre>org.testng.SkipException: Test is Skipped
	at testcases.CreateCustomerAPI.TC2_skipTestcase(CreateCustomerAPI.java:79)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:829)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>testcases.CreateCustomerAPI@333291e3</td></tr>
</table><p>
</body>
</html>